### ISSUE TITLE
Add option to conveniently filter-out channel from DeArrow from YouTube

### ISSUE THREAD

**Issue by micka190** [ajayyy/DeArrow#106]
There's quite a few channels I'm subscribed to that I would like to completely disable the extension for.
To do this, I need to manually create an Allowlist configuration, go to every channel's "About" page, and get their YouTube channel ID from the share button.

Would it be possible to simply have the option to add a channel to an Allowlist configuration *from* YouTube itself? Something like adding a dropdown menu to the DeArrow UI with a "Add channel to Allowlist" underneath it?

**Comment by whitespacecode** [comment]
Does the allowlist work for you?
I added one channel id but when i go to the channel the extension still applies changes

**Comment by B1ack3ye** [comment]
Absolutely, I'm on board with this suggestion, and I'd love to see it implemented. This idea sounds really convenient and would significantly simplify the use of the extension. Manually creating an Allowlist configuration for each channel can be time-consuming, especially if you have a lot of subscriptions. Adding a dropdown menu to the DeArrow user interface with an option to "Add channel to Allowlist" would be a fantastic enhancement. It would greatly enhance user-friendliness and contribute to making the extension even more efficient. I hope this feature gets implemented in the future!
